ext/drbbdup/drbbdup.c

```diff
@@ -48,20 +48,24 @@
 #    define LOG(dc, mask, level, ...)
 #endif
 
-/**
- * DynamoRIO Basic Block Duplication Extension: a code builder that
+/* DynamoRIO Basic Block Duplication Extension: a code builder that
  * duplicates code of basic blocks and dispatches control according to runtime conditions
  * so that different instrumentation may be efficiently executed.
  */
 
+#ifndef X86
+/* TODO i#4134: ARM not yet supported. */
+#    error
```

Comment:
Generally a message is given: `# error ARM is not yet supported`

Reply:
Done.